ethereum/circuits/lib/src/account_with_storage_recursive_int_test.nr

```diff
@@ -0,0 +1,16 @@
+use crate::account_with_storage_recursive::{get_account_with_storage_recursive, RecursiveProof};
+use crate::account_with_storage::StorageWithinBlock;
+use crate::fixtures::mainnet::paris::usdc_circle::{header::{number, hash}, account::{address, account}, storage::{keys, values}};
+use crate::chain::ETHEREUM_MAINNET_ID;
+use crate::misc::types::{Bytes32, Address};
+use dep::std::test::OracleMock;
+
+#[test]
+fn success() {
+    let result = StorageWithinBlock { block_hash: hash, account, values };
+    let recursive_proof = RecursiveProof { key_hash: 1, verification_key: [0; 114], proof: [0; 93] };
```

Comment:
Let's add a comment that this is not a correct proof but it passes because nargo does not verify it. Nargo thinks that it's bb's job, but doesn't call bb.
At the same time - key_hash can't be zero as other fields because nargo does some verification

Reply:
Maybe let's split it in two lines. First one assigning zeroed() and second - assigning key. That way we can split the message into two parts too.

Reply:
It seems that `key_hash` can also be zeroed and passes proof verification